ee/desktop/runner/runner_linux.go

```diff
@@ -91,6 +94,8 @@ func (r *DesktopUsersProcessesRunner) userEnvVars(uid string) map[string]string
 	}
 
 	sessionList := strings.Split(sessions, " ")
+
+CheckSessions:
```

Comment:
Whoa. Labels. Do we need this, or is this indicative of being too complex and that we should have some other functions?

Reply:
I think you're right and this could use a refactor, I'll give it another look!

Reply:
This is updated and I think it's more readable now 🙂